_posts/2024-05-03-trino-snowflake-bloomberg-oss-win.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title:  "Integrating Trino and Snowflake: An open source success story"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+canonical_url: https://bitsondata.dev/trino-snowflake-bloomberg-oss-win
```

Comment:
not published yet, make sure this link works before merging

Reply:
"This page is missing." During my test. Maybe because it's not published yet